README.md

```diff
@@ -1,11 +1,11 @@
-<p align="center"><a href="http://geomesa.github.io"><img align="center" width="50%" src="https://raw.githubusercontent.com/geomesa/geomesa.github.io/master/img/geomesa-2x.png"></img></a>
-<br/>
-<img align="center" height="150px" src="http://www.geomesa.org/img/geomesa-overview-848x250.png"></img>
-</p>
+<p align="center"><a href="http://geomesa.github.io"><img src="https://raw.githubusercontent.com/geomesa/geomesa.github.io/master/img/geomesa-2x.png"></img></a></p>
```

Comment:
Rebase to the latest on master and then run `mvn process-resources -N` to remove these changes.